relayer/src/index.ts

```diff
@@ -36,19 +37,43 @@ function getContractAddress(name: string): string {
     return require(`../build/contracts/${name}.json`).networks[NETWORK_CONFIGS.networkId].address.toLowerCase();
 }
 
+// Hardcoded Contract Addresses for the Thor Snapshot.
+// Just change `contractAddressesForTestnet` with `contractAddressesForDemo` in `contractWrappers` to use
+const DEMO_ADDRESSES = {
```

Comment:
We will need to add dummy `dutchAuction` addresses here since it's a requirement with 0x